core/src/main/resources/error/error-classes.json

```diff
@@ -5680,4 +5675,4 @@
       "Failed to get block <blockId>, which is not a shuffle block"
     ]
   }
-}
\ No newline at end of file
+}
```

Comment:
Revert this back, see https://github.com/apache/spark/pull/41256

Reply:
https://github.com/apache/spark/commit/32d42bbe98da9a7e8c38b9c3187c75dbbaaaafbb removed the last empty line.
This PR restores it.